cli_tools_tests/go.mod

```diff
@@ -1,15 +1,21 @@
-module github.com/GoogleCloudPlatform/compute-image-tools/cli_tools_integ_test/boot_inspect
+module github.com/GoogleCloudPlatform/compute-image-tools/cli_tools_tests
```

Comment:
I thought e2e and module tests will be separate packages? We don't need to have a code dependency on cli_tools in E2E tests, only module tests need it. This dependency can potentially lead to e2e tests picking up this dependency as well (possibly by mistake).

Reply:
Copying to https://github.com/GoogleCloudPlatform/compute-image-tools/pull/1381